Documentation/ceph-monitoring.md

```diff
@@ -196,4 +196,27 @@ spec:
 labels:
   monitoring:
     prometheus: k8s
-[...]
\ No newline at end of file
+[...]
+```
+
+## Prometheus to auto grow OSDs
+
+### To scale OSDs Vertically
+
+If you need to auto grow the size of OSDs on a PVC based rook-ceph cluster whenever the OSDs have reached the storage threshold you can use Prometheus for it.
+
+You need to just run the script `increase-size-of-osd.sh`.
+```console
+.\increase-size-of-osd.sh
```

Comment:
I'd suggest we have a single script that takes some arguments, something like this:
```
# for horizontal scaling
./auto-grow-storage.sh count <maxCount>
# for vertical scaling
./auto-grow-storage.sh size <maxSize>
```
With a single script we can re-use some helpers for alert manager, etc.